packages/beacon-node/src/chain/prepareNextSlot.ts

```diff
@@ -167,6 +161,7 @@ export class PrepareNextSlotScheduler {
             )) as CachedBeaconStateExecutions;
             updatedHeadRoot = proposerHeadRoot;
           }
+          this.computeStateHashTreeRoot(updatedPrepareState);
```

Comment:
do this last or after `prepareExecutionPayload()`, we want to let EL prepare payload as early as possible